index.html

```diff
@@ -12,11 +12,13 @@
     <script src="https://kit.fontawesome.com/yourcode.js" crossorigin="anonymous"></script>
 </head>
 <body>
+    <!-- navbar section -->
    <div class="container">
       <div class="heada">
             <div class="header-container">
-              <h2>Farmers Market</h2>
+              <h2>Farmers <span>Market</span></h2>
             </div>
+            <!-- shopping cart favicon -->
             <div>
                 <a href="#"><i class="fa-solid fa-cart-shopping" id="icon"></i></a>
```

Comment:
Add enough width to the side menu for mobile. Use a side menu for tablet too.

Reply:
should i use a media query to determine the screen size?